.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -397,4 +530,112 @@ sqs_GetQueueAttributes:
+          sdkguide: sdk-for-javascript/v2/developer-guide/sqs-examples-managing-visibility-timeout.html#sqs-examples-managing-visibility-timeout-setting
+          excerpts:
+            - description: Receive an &SQS; message and change its timeout visibility.
+              snippet_tags:
+                - sqs.JavaScript.visibility.receiveMessage
+  services:
+    sqs: {ChangeMessageVisibility}
+sqs_EnableLongPollingCreatingQueue:
+  title: Create an &SQS; queue that waits for a message
+  title_abbrev: Create a queue that waits for a message
+  synopsis: create an &SQS; queue that waits for a message to arrive.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/sqs
+          sdkguide: sdk-for-javascript/v3/developer-guide/sqs-examples-enable-long-polling.html#sqs-examples-enable-long-polling-existing-queue
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - sqs.JavaScript.createclientv3
+            - description: Create an &SQS; queue that waits for a message to arrive.
+              snippet_tags:
+                - sqs.JavaScript.longPoll.createQueueV3
+        - sdk_version: 2
+          github: javascript/example_code/sqs
+          sdkguide: sdk-for-javascript/v3/developer-guide/sqs-examples-enable-long-polling.html#sqs-examples-enable-long-polling-existing-queue
+          excerpts:
+            - description: Create an &SQS; queue that waits for a message to arrive.
+              snippet_tags:
+                - sqs.JavaScript.longPoll.createQueue
+  services:
+    sqs: {}
+sqs_SetQueueAttributes:
+  title: Change how long an &SQS; queue waits for a message
+  title_abbrev: Change how long a queue waits for a message
+  synopsis: change how long an &SQS; queue waits for a message to arrive.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/sqs
+          sdkguide: sdk-for-javascript/v3/developer-guide/sqs-examples-enable-long-polling.html#sqs-examples-enable-long-polling-existing-queue
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - sqs.JavaScript.createclientv3
+            - description: Change how long an &SQS; queue waits for a message to arrive.
+              snippet_tags:
+                - sqs.JavaScript.longPoll.setQueueAttributesV3
+        - sdk_version: 2
+          github: javascript/example_code/sqs
+          sdkguide: sdk-for-javascript/v3/developer-guide/sqs-examples-enable-long-polling.html#sqs-examples-enable-long-polling-existing-queue
+          excerpts:
+            - description: Change how long an &SQS; queue waits for a message to arrive.
+              snippet_tags:
+                - sqs.JavaScript.longPoll.setQueueAttributes
+  services:
+    sqs: {SetQueueAttributes}
+sqs_LongPollingReceiveMessage:

```

Comment:
This example feels more like a variant of `sqs_ReceiveMessage`, so I'd put it as a second set of excerpts under that instead of as a new example.

Reply:
Good point. I've reworked it slightly by changing the filename `sqs_ReceiveMessage` to `sqs_DeleteMessage`, as that's the purpose of the example, and added it to the `sqs_DeleteMessage` block; and I've added the metadata for `sqs_longpolling_receivemessages.js` to the `sqs_ReceiveMessages` block.